Source/Relativity.Transfer.SDK.Samples.Repository/Common/SampleOrder.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Relativity.Transfer.SDK.Samples.Repository.Common;
+
+public enum SampleOrder
```

Comment:
```suggestion
internal enum SampleOrder
```
Use `internal` where possible. If you have access denied for a type, use `InternalsVisibleTo` between projects.

Reply:
Done